src/ts/components/Controls.tsx

```diff
@@ -33,45 +32,58 @@ interface IState {
  */
 class Controls extends Component<IProps, IState> {
   public state = {
-    isOpen: false,
     allCategories: [],
     currentCategories: [],
     isLoadingCategories: false,
-    pluginState: undefined,
+    pluginState: undefined
   } as IState;
   public componentWillMount() {
```

Comment:
We should aim to remove componentWillMount whenever possible as it's now deprecated. https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html